packages/story-editor/src/components/panels/design/link/link.js

```diff
@@ -88,10 +118,7 @@ function LinkPanel({ selectedElements, pushUpdateForObject }) {
     getElementsInAttachmentArea(selectedElements).length > 0 &&
     currentPage?.pageAttachment?.url?.length > 0;
 
-  const defaultLink = useMemo(
-    () => createLink({ url: '', icon: null, desc: null }),
-    []
-  );
+  const defaultLink = useMemo(() => createLink({ icon: null, desc: null }), []);
```

Comment:
Why the change here?

Reply:
CreateLink, will default url to an empty string for you. So this not needed.